index.html

```diff
@@ -115,6 +115,37 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
Hi @Constantine-Rozoff, you need to change this comment 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->`
into 
`<!-- ________ YourName Contributor card START ________ -->`
Otherwise future contributors and the maintainers will not be able to identify your card in the code.